[2024-06-02T15:12:07.454+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-01T17:00:00+00:00 [queued]>
[2024-06-02T15:12:07.496+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-01T17:00:00+00:00 [queued]>
[2024-06-02T15:12:07.499+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-06-02T15:12:07.669+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_detect_anomalies_script> on 2024-06-01 17:00:00+00:00
[2024-06-02T15:12:07.689+0000] {standard_task_runner.py:60} INFO - Started process 8359 to run task
[2024-06-02T15:12:07.698+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'alert', 'run_detect_anomalies_script', 'scheduled__2024-06-01T17:00:00+00:00', '--job-id', '722', '--raw', '--subdir', 'DAGS_FOLDER/alert.py', '--cfg-path', '/tmp/tmpqqp98vxu']
[2024-06-02T15:12:07.706+0000] {standard_task_runner.py:88} INFO - Job 722: Subtask run_detect_anomalies_script
[2024-06-02T15:12:07.930+0000] {task_command.py:423} INFO - Running <TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-01T17:00:00+00:00 [running]> on host 6f8b2a20b5b0
[2024-06-02T15:12:08.318+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='alert' AIRFLOW_CTX_TASK_ID='run_detect_anomalies_script' AIRFLOW_CTX_EXECUTION_DATE='2024-06-01T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-01T17:00:00+00:00'
[2024-06-02T15:12:08.538+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-06-02T15:12:08.580+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=alert, task_id=run_detect_anomalies_script, execution_date=20240601T170000, start_date=20240602T151207, end_date=20240602T151208
[2024-06-02T15:12:08.729+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-06-02T15:12:09.014+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
